code/addons/docs/package.json

```diff
@@ -119,9 +119,11 @@
     "ts-dedent": "^2.0.0"
   },
   "devDependencies": {
+    "@rollup/pluginutils": "^5.0.2",
     "react": "^16.14.0",
     "react-dom": "^16.8.0",
-    "typescript": "~4.9.3"
+    "typescript": "~4.9.3",
+    "vite": "^4.0.4"
```

Comment:
@ndelangen can you confirm here that this means `@rollup/pluginutils` will get prebundled in, but `vite` and `typescript` won't, because they're not imported in the source?

It's roughly a [23kB increase in bundle size](https://bundlejs.com/?q=%40rollup%2Fpluginutils&treeshake=%5B%7B+createFilter+%7D%5D), but I don't see any other way, even source-copy is the same size

Reply:
Yes.